### DISCUSSION TITLE
Assign QSO for LoTW upload

### DISCUSSION THREAD

**Discussion by 4O4A** [magicbug/Cloudlog#1425] [Q&A] [answered]
Hi all,

First of all -- great product. I have been looking for some online log for a long time and friend of mine recommended this for me. 
Works very nice, easy to setup on my server, etc. 
I setup cron jobs for upload/download, etc. Everything works fine.

However, one thing puzzles me.

When I manually add QSO, I don't see that I can queue it for LoTW upload? 
I know how to edit QSO and assign put it in queue for next crone job driven upload to LoTW, but how to that at the moment when I fill the QSO data?

Regards,
Dragan, 4O4A

**Comment by magicbug** [comment, marked as answer]
If you manually add a qso via POST QSO it will schedule it for upload, if you edit a QSO and need to upload it again go to "QSL" tab then LoTW then change "Sent" to "no" and it will reupload it.

**Reply by 4O4A** [reply]
Thanks, Peter.  "Queued" confused me a little bit. Great work, besides! Thanks.
